### PR TITLE
feat: add Usage Control structure

### PR DESCRIPTION
**Pull request by sebbader-sap**
To get the discussion about the EDC Usage Control view sorted, this PR
adds a number of first _viewpoints_.

Related to https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions/1229 and https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1406

### CODE REVIEW

**Comment by sebbader-sap** [comment]
@juliapampus this is the PR as discussed in our call today morning. (cannot assign you directly)

**Comment by juliapampus** [comment]
@sebbader-sap Updated the repo

File: Work Content/Usage Control/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- [Transparency](transparency.md) (TODO)
+- [Temper Resistance](temper_resistance.md) (TODO)
+- [Enforcement](enforcement.md) (TODO)
+- ...
+
+## Overview
+
+Usage Control is a concept to expand data protection from the provider system to the IT environment of the user. Similar to Security, Usage Control is never absolute but depends on the requirements and threads of each use case. Also similar to Security, Usage Control requires a comprehensive treatment as the overall control level is only as strong as its weakest aspect.
+
+Due to the complexity of the topic and the huge amount of influencing factors, a matrix structure is outlined how to organise the various aspects, shown below. The respective aspects are grouped in _viewpoints_, each regarding a dedicated set of _concerns_, which have been introduced through identified _stakeholders_. The approach follows roughly the specifications of ISO/IEC 42010 and uses its terminology in the following.
```
Comment (juliapampus) [inline Work Content/Usage Control/README.md:18]: ISO/IEC 42010 is missing a link

File: Work Content/Usage Control/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Usage Control
+
+## Content:
```
Comment (juliapampus) [inline Work Content/Usage Control/README.md:3]:
```suggestion
```

File: Work Content/Usage Control/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- [IAM](iam.md) (TODO)
+- [Access Control](access_control.md) (TODO)
+- [Transparency](transparency.md) (TODO)
+- [Temper Resistance](temper_resistance.md) (TODO)
+- [Enforcement](enforcement.md) (TODO)
+- ...
+
+## Overview
+
+Usage Control is a concept to expand data protection from the provider system to the IT environment of the user. Similar to Security, Usage Control is never absolute but depends on the requirements and threads of each use case. Also similar to Security, Usage Control requires a comprehensive treatment as the overall control level is only as strong as its weakest aspect.
```
Comment (juliapampus) [inline Work Content/Usage Control/README.md:16]: I wouldn't capitalize the terms, e.g., "Security" and "Usage Control".

**Comment by sebbader-sap** [comment]
#3 overrules this PR, the new one contains way more changes.

**Pull request closed without merging**